### PR TITLE
DEV-3899 add subject, test and solution for the exercise change-struct

### PR DESCRIPTION
**Pull request by MSilva95**

### CODE REVIEW

File: subjects/change-struct/README.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+```console
+User-> tar -cf done.tar *
+User-> ls new-struct/
+0_to_3  4  6_to_9  A  done.tar
+```
+
+**Only `done.tar` should be submitted.**
+
+**Tips:**
```
Comment (zamazzal) [inline subjects/change-struct/README.md:77]: tips should be in H3

**Review by zamazzal — CHANGES_REQUESTED** [review]

**Comment by zamazzal** [comment]
you have to move the subject folder inside subject/devops folder

File: subjects/change-struct/README.md (hunk 1 above)
Comment (MSilva95) [reply subjects/change-struct/README.md]: Done

**Review by zamazzal — CHANGES_REQUESTED** [review]
i can't understand why you have this folders inside the 'solutions' folder:

- 0_to_3
- 4
- 6_to_9
- A

**Review by zamazzal — APPROVED** [review]

**Pull request merged**